client/app/telecom/overTheBox/tasks/overTheBox-tasks.html

```diff
@@ -9,77 +9,21 @@
-                            <td data-title="{{:: 'overTheBox_tasks_name' | translate}}"
-                                data-ng-bind="task.name">
-                            </td>
-                            <td data-title="{{:: 'overTheBox_tasks_id' | translate}}"
-                                data-ng-bind="task.taskId">
-                            </td>
-                            <td class="text-right"
-                                data-title="{{:: 'overTheBox_tasks_status' | translate}}">
-                                <span class="label"
-                                      data-ng-class="{
-                                          'label-success': task.status === 'done',
-                                          'label-danger': tasks.status === 'error',
-                                          'label-info': tasks.status === 'doing',
-                                          'label-primary': tasks.status === 'todo'
-                                      }">
-                                    {{ 'overTheBox_tasks_status_' + task.status | translate }}
-                                </span>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-            <div class="mt-2"
-                 data-pagination-front
-                 data-go-to-page="false"
-                 data-items="OverTheBoxTasks.taskIds"
-                 data-paginated-items="OverTheBoxTasks.tasks"
-                 data-current-page="currentPage"
-                 data-nb-pages="maxSize"
-                 data-items-per-page="OverTheBoxTasks.filter.perPage"
-                 data-transform-item="OverTheBoxTasks.onTransformItem(item)"
-                 data-page-placeholder="{{ 'common_pagination_go_to_page' | translate }}"
-                 data-item-per-page-placeholder="{{'common_pagination_show_by_page' |translate }}">
-            </div>
+            <oui-datagrid data-rows="OverTheBoxTasks.taskIds" data-row-loader="OverTheBoxTasks.transformItem($row)">
+                <oui-column data-title="'overTheBox_tasks_name' | translate" data-property="name" data-sortable></oui-column>
+                <oui-column data-title="'overTheBox_tasks_id' | translate" data-property="taskId" data-sortable></oui-column>
+                <oui-column data-title="'overTheBox_tasks_status' | translate" data-property="status" data-sortable="asc">
+                    <span class="label"
+                          data-ng-class="{
+                                'label-success': $row.status === 'done',
+                                'label-danger': $row.status === 'error',
+                                'label-info': $row.status === 'doing',
+                                'label-primary': $row.status === 'todo'
+                            }">
+                        {{ 'overTheBox_tasks_status_' + $row.status | translate }}
```

Comment:
data-ng-bind is normally used in those cases

Reply:
or `data-translate directly` (optional)

Other thing, I'm not sure that `status` should be the default sorting here... in my opinion!